flexmeasures/api/sunset/__init__.py

```diff
@@ -0,0 +1,69 @@
+from flask import Flask, Blueprint
+
+from flexmeasures.api.common.utils.deprecation_utils import (
+    deprecate_blueprint,
+    sunset_blueprint,
+)
+
+
+# The sunset API blueprints. They are registered with the Flask app (see register_at)
+flexmeasures_api_v1 = Blueprint("flexmeasures_api_v1", __name__)
+flexmeasures_api_v1_1 = Blueprint("flexmeasures_api_v1_1", __name__)
+flexmeasures_api_v1_2 = Blueprint("flexmeasures_api_v1_2", __name__)
+flexmeasures_api_v1_3 = Blueprint("flexmeasures_api_v1_3", __name__)
+flexmeasures_api_v2_0 = Blueprint("flexmeasures_api_v2_0", __name__)
+
+SUNSET_INFO = [
+    dict(
+        blueprint=flexmeasures_api_v1,
+        api_version_sunset="1.0",
+        sunset_link="https://flexmeasures.readthedocs.io/en/v0.13.0/api/v1.html",
+    ),
+    dict(
+        blueprint=flexmeasures_api_v1_1,
+        api_version_sunset="1.1",
+        sunset_link="https://flexmeasures.readthedocs.io/en/v0.13.0/api/v1_1.html",
+    ),
+    dict(
+        blueprint=flexmeasures_api_v1_2,
+        api_version_sunset="1.2",
+        sunset_link="https://flexmeasures.readthedocs.io/en/v0.13.0/api/v1_2.html",
+    ),
+    dict(
+        blueprint=flexmeasures_api_v1_3,
+        api_version_sunset="1.3",
+        sunset_link="https://flexmeasures.readthedocs.io/en/v0.13.0/api/v1_3.html",
+    ),
+    dict(
+        blueprint=flexmeasures_api_v2_0,
+        api_version_sunset="2.0",
+        sunset_link="https://flexmeasures.readthedocs.io/en/v0.13.0/api/v2_0.html",
+    ),
+]
+
+for info in SUNSET_INFO:
+    deprecate_blueprint(
+        blueprint=info["blueprint"],
+        deprecation_date="2022-12-14",
```

Comment:
I'd expect the dates to be in the `info`dict, as there will be others later. But I understand if you think this csan be done when that actually happens.